packages/dd-trace/src/appsec/iast/analyzers/sql-injection-analyzer.js

```diff
@@ -6,7 +6,7 @@ class SqlInjectionAnalyzer extends InjectionAnalyzer {
     super('SQL_INJECTION')
     this.addSub('apm:mysql:query:start', ({ sql }) => this.analyze(sql))
     this.addSub('apm:mysql2:query:start', ({ sql }) => this.analyze(sql))
-    this.addSub('apm:pg:query:start', ({ statement }) => this.analyze(statement))
+    this.addSub('apm:pg:query:start', ({ query }) => query && this.analyze(query.text))
```

Comment:
Is this not tested? 🤔

Reply:
I didn't know that we have a way to do integration test using the real plugins. I've created new PR, adding a way to apply plugin test to iast related test:

https://github.com/DataDog/dd-trace-js/pull/2565